AppleTVRemoteDemoApp.xcodeproj/project.pbxproj

```diff
@@ -90,6 +90,7 @@
 				TargetAttributes = {
 					4FCB9BC41BCC608D00B4F841 = {
 						CreatedOnToolsVersion = 7.1;
+						DevelopmentTeam = 268S5GJFWZ;
```

Comment:
Is this project change needed?

Reply:
@lucianomarisi - no it's not, I'll revert that, I was testing on my TV device.